timemachine/fe/chiral_utils.py

```diff
@@ -145,7 +145,7 @@ def find_chiral_atoms(mol):
     chiral_patterns = [
         "[X4:1]",  # any tetrahedral atom
         "[#16X3,#15X3:1]",  # trivalent sulfur, phosphorous are assumed to be non-invertible
-        # "[#7X3:1](~[R])(~[R])~[R]",  # nitrogen directly bonded to three ring atoms  # TODO: handle pyramidal nitrogen
+        "[#7X3:1](~[#9])(~[#9])~[#9]",  # nitrogen directly bonded to three ring atoms  # TODO: handle pyramidal nitrogen
```

Comment:
reverted

Reply:
re-reverted - I had to add back in the NF3 pattern for some of the tests (namely, `test_interpolate.py::test_core_chiral_interpolation_sp2_to_invertible_sp3_fwd`).

I originally added this so I can easily create a test case for a non-invertible 3-connected pyramidal group (without resorting to 4-connected groups). This pattern is exact, i.e. it will *only* match the NF3 molecule exactly (it's a highly volatile compound with no practical purposes).

Reply:
tagging @maxentile for visibility. Added a comment specifying the intended usage.